### ISSUE TITLE
[Adaptive Card-Schema Explorer]: Visual label is missing on the adaptive card edit field.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7914]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)

Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'Input.Number" link present under Input from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards using the Tab key.
5. Navigate to the number edit field present under Adaptive Card using Tab key.
6. Observe the issue here.

Actual Result:
visually Label name is missing on the Edit field.

Expected Result:
Label name has to be given on Number edit box as "Enter a Number".

Note:
Same issue repro with What is your first name and what is your last name under "Action.Submit" control.
User Experience:
Keyboard users and cognitive users will get impacted if label is not given for the Date and Time Icons.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/03.03.02

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="960" alt="Visual name is missing on the edit-number" src="https://user-images.githubusercontent.com/104482612/191669941-a2f52322-6138-45bb-a3f1-4ff9d928e0c1.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is still repro on the environment below, hence reactivating this issue.

**Test Environment:**
URL: https://adaptivecards.io/explorer/Input.Number.html
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1

**Snippet:**
![image](https://user-images.githubusercontent.com/95913748/234814691-9e038b91-4062-4654-8365-06f6a59e68bd.png)

**Comment by vagpt** [comment]
We will validate this issue by the next week as the PR was merged on 4/27/2023.

**Comment by vagpt** [comment]
This issue is now working fine, hence adding A11yTTValidated.

![Image](https://github.com/user-attachments/assets/901af453-7f2a-4b53-9610-b2b643bf0982)
